iam-login-service/src/main/java/it/infn/mw/iam/api/tokens/service/TokenService.java

```diff
@@ -0,0 +1,16 @@
+package it.infn.mw.iam.api.tokens.service;
+
+import it.infn.mw.iam.api.tokens.exception.TokenNotFoundException;
+import it.infn.mw.iam.api.tokens.model.TokensListResponse;
+import it.infn.mw.iam.api.tokens.service.filtering.TokensFilterRequest;
+import it.infn.mw.iam.api.tokens.service.paging.TokensPageRequest;
+
+public interface TokenService<T> {
+
+  TokensListResponse<T> getList(final TokensPageRequest pageRequest, final TokensFilterRequest filterRequest);
```

Comment:
getList -> getTokensForClient, getTokensForUser, getTokensForClientAndUser, getAllTokens and drop the TokensFilterRequest object